packages/core/src/tools/error.spec.ts

```diff
@@ -1,8 +1,23 @@
 import type { StackTrace } from '../domain/tracekit'
-import { createHandlingStack, formatUnknownError, getFileFromStackTraceString } from './error'
-
-describe('formatUnknownError', () => {
+import type { RawErrorCause, ErrorWithCause } from './error'
+import { clocksNow } from './timeUtils'
+import {
+  createHandlingStack,
+  computeRawError,
+  getFileFromStackTraceString,
+  flattenErrorCauses,
+  ErrorSource,
+  ErrorHandling,
+} from './error'
+
+describe('computeRawError', () => {
   const NOT_COMPUTED_STACK_TRACE: StackTrace = { name: undefined, message: undefined, stack: [] } as any
+  const DEFAULT_RAW_ERROR_PARMS = {
+    originalError: undefined,
+    startClocks: clocksNow(),
+    nonErrorPrefix: 'Uncaught',
+    source: ErrorSource.CUSTOM,
+  }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏always set
```suggestion
  const DEFAULT_RAW_ERROR_PARMS = {
    startClocks: clocksNow(),
    nonErrorPrefix: 'Uncaught',
    source: ErrorSource.CUSTOM,
  }
```